contracts/multisig-prover/src/encoding/bcs.rs

```diff
@@ -0,0 +1,77 @@
+use axelar_wasm_std::operators::Operators;
+use bcs::to_bytes;
+use cosmwasm_std::{HexBinary, Uint256};
+
+use crate::{error::ContractError, state::WorkerSet};
+
+pub fn make_operators(worker_set: WorkerSet) -> Operators {
+    let mut operators: Vec<(HexBinary, Uint256)> = worker_set
+        .signers
+        .iter()
+        .map(|s| (s.pub_key.clone().into(), s.weight))
+        .collect();
+    operators.sort_by_key(|op| op.0.clone());
+    Operators {
+        weights_by_addresses: operators,
+        threshold: worker_set.threshold,
+    }
+}
+
+fn u256_to_u128(val: Uint256) -> u128 {
+    u128::from_le_bytes(
+        val.to_le_bytes()[..16]
```

Comment:
Validate the length. Simply ignoring other bytes makes me unconfortable.

Reply:
didn't we have this in another PR (maybe it was different integer sizes). Seems like the asserts you've been using would be appropriate here as well

Reply:
We have this in another PR, and it does validate the length there. I can just change this when merging